bundles/org.openhab.binding.sensibo/src/main/java/org/openhab/binding/sensibo/internal/dto/settimer/SetTimerRequest.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sensibo.internal.dto.settimer;
+
+import org.eclipse.jetty.http.HttpMethod;
+import org.openhab.binding.sensibo.internal.dto.AbstractRequest;
+import org.openhab.binding.sensibo.internal.dto.poddetails.AcStateDTO;
+
+/**
+ * All classes in the ..binding.sensibo.dto are data transfer classes used by the GSON mapper. This class reflects a
+ * part of a request/response data structure.
+ *
+ * @author Arne Seime - Initial contribution.
+ */
+
+public class SetTimerRequest extends AbstractRequest {
```

Comment:
I think adding a DTO suffix remove the null warning for this class.

Reply:
What do you mean?

Reply:
Changing the class name from `SetTimerRequest` to `SetTimerRequestDTO` will exclude the class from null check warnings.

Reply:
I added https://github.com/openhab/openhab-addons/pull/7257 to resolve it instead